### PR TITLE
DCS-495: code refactor:

### PR DESCRIPTION
**Pull request by GurnankCheema**
getLocation() and getIncidentLocations() moved from  offenderService to LocationService

### CODE REVIEW

File: server/services/offenderService.test.ts [hunk 1]
```diff
@@ -133,7 +137,7 @@ describe('getOffenders', () => {
         { id: 4, userDescription: 'place 4' },
       ])
 
-      const result = await service.getIncidentLocations(token, 'WRI')
+      const result = await locationService.getIncidentLocations(token, 'WRI')
```
Comment (andrewrlee) [inline server/services/offenderService.test.ts:140]: these tests are now in the wrong file - this is testing the location service not the offender service

File: server/services/locationService.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import { Prison } from '../types/uof'
+import { AgencyId, LocationService, Prison, PrisonLocation } from '../types/uof'
 
 import logger = require('../../log.js')
 
-export default function createLocationService(elite2ClientBuilder, incidentClient) {
+export default function createLocationService(elite2ClientBuilder, incidentClient): LocationService {
```
Comment (andrewrlee) [inline server/services/locationService.ts:5]: 👍

File: server/services/offenderService.test.ts (hunk 1 above)
Comment (GurnankCheema) [reply server/services/offenderService.test.ts:140]: You are correct.
Have moved these tests to locationService.test

File: server/services/locationService.ts [hunk 3]
```diff
@@ -16,14 +16,44 @@ export default function createLocationService(elite2ClientBuilder, incidentClien
...
+      logger.error(error, 'Error during getIncidentLocations')
+      throw error
+    }
   }
 
   return {
     getPrisons,
     getPrisonById,
-    updateAgencyId,
+    // updateAgencyId,
```
Comment (andrewrlee) [inline server/services/locationService.ts:55]: can remove this comment

**Pull request merged**